root/programs/C#/z_Common2.bat

```diff
@@ -3,13 +3,20 @@
 @rem --------------------------------------------------
 
 @rem --------------------------------------------------
+@rem Set Program Files path
+@rem --------------------------------------------------
+reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set PROGRAM_FILES=C:\Program Files || set PROGRAM_FILES=C:\Program Files (x86)
```

Comment:
@daisukenishino 

You should modify this code.
Please see [here](https://github.com/OpenTouryoProject/OpenTouryoTemplates/pull/24#discussion_r48399310).